contracts/libs/0.8.x/BytecodeStorage.sol

```diff
@@ -0,0 +1,183 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+pragma solidity ^0.8.0;
+
+/**
+ * @title Art Blocks Script Storage Library
+ * @author Art Blocks Inc.
+ * @author Modified from 0xSequence (https://github.com/0xsequence/sstore2/blob/master/contracts/SSTORE2.sol)
+ * @author Modified from Solmate (https://github.com/transmissions11/solmate/blob/main/src/utils/SSTORE2.sol)
```

Comment:
TODO: I'm going to add a quick explanation here under these that clarifies what modifications this library makes relative to the two SSTORE2 libraries above?

Reply:
This has also been addressed, and documentation cleaned up with a couple more passes.